pkg/controller/certificates/certificate_controller.go

```diff
@@ -162,13 +165,14 @@ func (cc *CertificateController) enqueueCertificateRequest(obj interface{}) {
 }
 
 func (cc *CertificateController) syncFunc(ctx context.Context, key string) error {
+	logger := klog.FromContext(ctx)
 	startTime := time.Now()
 	defer func() {
-		klog.V(4).Infof("Finished syncing certificate request %q (%v)", key, time.Since(startTime))
+		logger.V(4).Info("Finished syncing certificate request", "certificate", key, "elapsedTime", time.Since(startTime))
 	}()
 	csr, err := cc.csrLister.Get(key)
 	if errors.IsNotFound(err) {
-		klog.V(3).Infof("csr has been deleted: %v", key)
+		logger.V(3).Info("csr has been deleted", "csr", key)
```

Comment:
Use "csr" for `CertificateSigningRequest` objects. 
```suggestion
		logger.V(3).Info("csr has been deleted", "key", key)
```

Reply:
I think there should not be much difference between the key and the object since it symbolizes the same thing. The value should render even in the same way. 
I think we can name these csr as well since key does not have much meaning to the k8s user.